src/main/java/plansoft/skillsmanagement/level/LevelService.java

```diff
@@ -24,8 +24,21 @@
 	}
 
 	public void addLevel(Level level) {
+		List<Level> levels = new ArrayList<>();
```

Comment:
@LORYMAX4 questo metodo ha bisogno di un refactor.. mi spieghi a parole cosa dovrebbe fare questo metodo?

Reply:
Questo metodo permette l'aggiunta di un nuovo livello, nello specifico mi creo una lista dove al suo interno ci sono tutti i livelli già presenti nel database, successivamente mi vado a scorrere tutta la lista andando a cercare se esiste nel database un livello che ha lo stesso levelOrder di quello che ho in input, in caso non ci fosse aggiungo il nuovo livello, se invece è già presente un livello con lo stesso levelOrder, allora mi salvo quel livello e poi vado ad incrementare di 1 il levelOrder partendo dal livello nel quale il levelOrder era uguale a quello dato in input